ValerieRono/ireporter
# #162323705 delete specific record

#### What does this PR do?
allow a user to delete a speicifc record

#### Description of Task to be completed?
create delete method, add to resource

#### How should this be manually tested?
follow instructions in read me file and ensure app is running
using postman, copy url from terminal(refer to readme), insert /incidents.<int:id> after the url, where <int:id> is the id of the record you want to delete

#### Any background context you want to provide?
used flask marshmallow and flask restful

#### What are the relevant pivotal tracker stories?
#162323705

#### Screenshots (if appropriate)
![screenshot from 2018-11-30 10-47-55](https://user-images.githubusercontent.com/40663828/49275788-855f9500-f48d-11e8-93ce-e8029b928044.png)
![screenshot from 2018-11-30 10-48-16](https://user-images.githubusercontent.com/40663828/49275795-8d1f3980-f48d-11e8-92fc-b3b5909874e3.png)
![screenshot from 2018-11-30 10-48-29](https://user-images.githubusercontent.com/40663828/49275805-94464780-f48d-11e8-9ec5-aef0398f93e7.png)
